src/lib/UI/Config/Provider/Languages.php

```diff
@@ -34,10 +38,12 @@ class Languages implements ProviderInterface
     public function __construct(
         LanguageService $languageService,
         ConfigResolverInterface $configResolver,
+        SiteAccess $siteAccess,
         array $siteAccesses
     ) {
         $this->languageService = $languageService;
         $this->configResolver = $configResolver;
+        $this->siteAccess = $siteAccess;
```

Comment:
Current SiteAccess shouldn't be taken from `SiteAccessService::getCurrent`?

Reply:
in most cases, from what I checked the code, we use eZ\Publish\Core\MVC\Symfony\SiteAccess, but if you think that we should use SiteAccessService, then I don't see a problem to change to SiteAccessService

Reply:
In v2.5 there is no `SiteAccessService::getCurrent()` but it will be used when creating PR for v3

Reply:
SiteAccessService is available since v3.0